contracts/foundry.toml

```diff
@@ -2,7 +2,7 @@
 src = 'src'
 out = 'out'
 libs = ['lib']
-solc_version = '0.8.18'
+solc_version = '0.8.20'
```

Comment:
Looks like the solidity releases, I think we can go up to `0.8.23` before any new opcodes are introduced in the default EVM version. I think it would make sense to use the latest possible version here while we're updating it.